src/components/Header.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { Box, Paper } from "@mui/material";
+import { BottomNavigation, BottomNavigationAction } from "@mui/material";
+import HomeIcon from "@mui/icons-material/Home";
+import HelpIcon from "@mui/icons-material/Help";
+import EventIcon from "@mui/icons-material/Event";
+
+const Header = () => {
+  return (
+    <Box sx={{ pb: 7 }}>
+      <Paper
+        sx={{ position: "fixed", bottom: 0, left: 0, right: 0 }}
+        elevation={3}
+        color="dark"
+      >
+        <BottomNavigation showLabels>
+          <BottomNavigationAction
+            component={Link}
```

Comment:
Did you have the navigation at the bottom of the page to make it mobile friendly? If so that's cool, think it looks good there, but for desktop I don't think it's great UX having it at the bottom.
Could you set it up to read the screen width and then place the navigation accordingly?

Reply:
This has been implemented